salt/metalk8s/orchestrate/bootstrap/init.sls

```diff
@@ -90,6 +90,15 @@ Bring bootstrap minion to highstate:
     - salt: Sync bootstrap minion
     - salt: Deploy CA role on bootstrap minion
 
+Generate etcd client certs for salt master:
+  salt.state:
+  - sls:
+    - metalk8s.salt.master.certs
+    - metalk8s.kubernetes.ca.etcd.advertised
```

Comment:
Why? This should be the case already by virtue of `Deploy CA role on bootstrap minion`. You should `require` that stage to be done before this one can run.

Reply:
Ok, I will fix